src/log_parser.py

```diff
@@ -256,6 +257,7 @@ def parse(ifs):
 
     """
     for line in ifs:
+        line = str(line).replace("\\r\\n\'", "") # garbage is getting added to the lines?
```

Comment:
Can you post what sort of garbage is getting added to the lines? I will need to confirm this doesn't break windows/linux before approving